build-ignore.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+[ "$SITE_NAME" == "pharos-storybooks" ] && git diff --quiet HEAD^ HEAD packages/pharos/ .storybook/ package.json yarn.lock netlify.toml || [ "$SITE_NAME" == "pharos" ] && git diff --quiet HEAD^ HEAD packages/pharos-site/ package.json yarn.lock netlify.toml
```

Comment:
Should be able to split this into a few lines, though I'm not sure about the syntax.